### ISSUE TITLE
Do some automated accessibility testing as part of CI

### ISSUE THREAD

**Issue by mlandauer** [digitalnsw/buy-nsw#56]
I'm certainly of the belief that automated testing is no replacement for good accessibility testing with real people but we can probably make our development lives easier if we keep a watch on any particular bad accessibility transgressions by using some automated testing as part of our CI workflow.

I'm in two minds as to whether accessibility problems should always cause a build to fail. I think, initially at least, we should be able to keep moving even if there are known accessibility problems but we keep getting notified and annoyed by them so they don't just get ignored.

The other part of me thinks that this is worth doing right from the beginning.
